lib/gooddata/helpers/csv_helper.rb

```diff
@@ -55,6 +68,20 @@ def write(opts, &_block)
             end
           end
         end
+
+        # Ammend a hash to CSV in a smart manner
+        # @option filename String
+        # @option data Hash
+        def ammend_line(filename, data)
+          current_data = read_as_hash(filename)
+          data_to_write = (current_data << data).map(&:sort).map { |r| Hash[r] }
```

Comment:
https://stackoverflow.com/questions/27384967/how-to-sort-a-ruby-hash-alphabetically-by-keys

since hashes return keys in the order of construction, I wanted to make sure no changes in the codebase would result in a CSV where one line does not match the headers because it was produced by a different logic.